application/features/bootstrap/UserContext.php

```diff
@@ -44,7 +44,7 @@ public function theUserStoreIsEmpty()
     }
 
     /**
-     * @When /^I request the user be (created|updated|deleted|retrieved|<.*>)$/
+     * @When /^I request the user be (created|updated|deleted|retrieved|headed|patched|<.*>)$/
```

Comment:
That's a great point...I simply did that to match the angled brackets for use in Scenario Outlines...maybe there's a better regexp for it, something to "optionally" match the angled brackets.